Modules/mathmodule.c

```diff
@@ -2009,12 +2009,11 @@ math.factorial
 
 Find n!.
 
```

Comment:
```suggestion
```

nit

Reply:
(Don't forgot to run clinic.py on that.)